scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -87,28 +87,35 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
         logger.setLevel(logging.ERROR)
 
     # Calculate the tools and game build paths
-    default_base_build_path = ctx.engine_path / 'build' if engine_centric else ctx.project_path / 'build'
+    default_base_path = ctx.engine_path if engine_centric else ctx.project_path
     
     if is_installer_sdk:
-            # Check if we have any monolithic entries if building monolithic
-            monolithic_artifacts = glob.glob(f'cmake/Platform/{exp.get_platform_installer_folder_name()}/Monolithic/ConfigurationTypes_*.cmake',
-                      root_dir=ctx.engine_path)
-            has_monolithic_artifacts = len(monolithic_artifacts) > 0
-
-            if monolithic_build and not has_monolithic_artifacts:
-                logger.error("Trying to create monolithic build, but no monolithic artifacts are detected in the engine installation! Please re-run the script with '-nomonolithic' and '-config profile'.")
-                return 1
-            tools_build_path = ctx.engine_path / 'bin' / exp.get_platform_installer_folder_name(selected_platform) / tool_config / 'Default'
+        # Check if we have any monolithic entries if building monolithic
+        if monolithic_build and not exp.has_monolithic_artifacts(ctx):
+            logger.error("Trying to create monolithic build, but no monolithic artifacts are detected in the engine installation! Please re-run the script with '-nomonolithic' and '-config profile'.")
```

Comment:
Wrong argument name. It should be `-nomono` or `--non-monolithic` (look at line 380)